test/build/cdn/expected-build/_dist_/index.js

```diff
@@ -3,8 +3,8 @@ import.meta.env = __SNOWPACK_ENV__;
 
 import React from "https://cdn.pika.dev/react@^16.13.1";
 import ReactDOM from "https://cdn.pika.dev/react-dom@^16.13.1";
-const App = () => React.createElement("div", null, "I’m an app!");
-ReactDOM.render(React.createElement(App, null), document.getElementById("root"));
+const App = () => /* @__PURE__ */ React.createElement("div", null, "I’m an app!");
+ReactDOM.render(/* @__PURE__ */ React.createElement(App, null), document.getElementById("root"));
```

Comment:
This is an expected diff from esbuild update (would have happened in any PR where this updated)